enterprise_catalog/apps/jobs/management/commands/process_enterprise_jobs.py

```
@@ -0,0 +1,67 @@
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command for associating jobs with enterprises
+
+    Example Usage:
+    >> python manage.py process_enterprise_jobs
+    """
+    help = (
+        'Associate jobs with enterprises'
+    )
+
+    def handle(self, *args, **options):
+        """
+        Associate jobs with enterprises
+        """
+        logger.info("Generating enterprise job association...")
+        algolia_client = get_initialized_algolia_client()
+        enterprise_uuids = set()
+        enterprise_catalogs = EnterpriseCatalog.objects.all()
+        for enterprise_catalog in enterprise_catalogs:
+            enterprise_uuids.add(enterprise_catalog.enterprise_uuid)
+
+        jobs = Job.objects.all()
+        for enterprise_uuid in enterprise_uuids:
+            for job in jobs:
+                try:
+                    job_skills = JobSkill.objects.filter(job=job).order_by('-significance')[:3]  # Get top 3 skills
+                    search_query = {
+                        'filters': f'(skill_names:{job_skills[0].name} OR \
+                            skill_names:{job_skills[1].name} OR \
+                            skill_names:{job_skills[2]}.name) AND \
+                            enterprise_customer_uuids:{enterprise_uuid}',
+                        'maxFacetHits': 50
+                    }
+                    response = algolia_client.algolia_index.search_for_facet_values('skill_names', '', search_query)
+                    for hit in response.get('facetHits', []):
+                        if hit.get('count') > 1:
+                            JobEnterprise.objects.update_or_create(
+                                job=job,
+                                enterprise_uuid=enterprise_uuid
+                            )
+                            break
+                except Exception as ex:  # pylint: disable=broad-exception-caught
+                    logger.error(
```

Shouldn't we allow it to fail loudly? If an error occurs, no one will know that something went wrong.